### PR TITLE
Correct documentation for support barycenter

### PR DESCRIPTION
**Pull request by AdrienCorenflos**
## Types of changes

- [ x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and context / Related issue
  

https://github.com/PythonOT/POT/issues/200

## How has this been tested (if it applies)
 
I can't render the doc locally so not tested

## Checklist

- [ x] The documentation is up-to-date with the changes I made.
- [x ] I have read the [**CONTRIBUTING**](CONTRIBUTING.md) document.
- [ ] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

**Comment by rflamary** [comment]
hello @AdrienCorenflos 

This is the output of the doc for your PR:
https://317-71472695-gh.circle-artifacts.com/0/dev/index.html

**Comment by AdrienCorenflos** [comment]
Looks good doesn't it?

https://317-71472695-gh.circle-artifacts.com/0/dev/gen_modules/ot.lp.html#ot.lp.free_support_barycenter

Adrien

Le mer. 15 juil. 2020 à 10:18, Rémi Flamary <notifications@github.com> a
écrit :

File: ot/lp/__init__.py [hunk 1]
```diff
@@ -443,18 +443,30 @@ def free_support_barycenter(measures_locations, measures_weights, X_init, b=None
     - we do not optimize over the weights
     - we do not do line search for the locations updates, we use i.e. theta = 1 in [1] (Algorithm 2). This can be seen as a discrete implementation of the fixed-point algorithm of [2] proposed in the continuous setting.
 
+    Formally we solve the following problem:
```
Comment (rflamary) [inline ot/lp/__init__.py:446]: Could you please put the optimization problem after the first sentence and before the discussion on the algorithm?

Also the itemize in the discussion does not work (it needs an empty line maybe)

**Comment by AdrienCorenflos** [comment]
Actually I'm gonna deleter this if it's ok:
"The function solves the Wasserstein barycenter problem when the barycenter
measure is constrained to be supported on k atoms"

It's redundant with the first sentence.


Le mer. 15 juil. 2020 à 16:01, Rémi Flamary <notifications@github.com> a
écrit :
>
> @rflamary commented on this pull request.
>
> ________________________________
>
> In ot/lp/__init__.py:
>
> > @@ -443,18 +443,30 @@ def free_support_barycenter(measures_locations,
measures_weights, X_init, b=None
>      - we do not optimize over the weights
>      - we do not do line search for the locations updates, we use i.e.
theta = 1 in [1] (Algorithm 2). This can be seen as a discrete
implementation of the fixed-point algorithm of [2] proposed in the
continuous setting.
>
> +    Formally we solve the following problem:
>
> Could you please put the optimization problem after the first sentence
and before the discussion on the algorithm?
>
> Also the itemize in the discussion does not work (it needs an empty line
maybe)

**Comment by rflamary** [comment]
Yes you can remove the repetition.

**Pull request merged**